sycl/test/warnings/warnings.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -Wall -Wpessimizing-move -Wunused-variable -Wmismatched-tags -Wunneeded-internal-declaration -Werror -Wno-unknown-cuda-version -fsycl %s -o %t.out
+// RUN: %clangxx -I /localdisk2/icl/fadeeval/sycl_workspace/build/include/sycl -Wall -Wpessimizing-move -Wunused-variable -Wmismatched-tags -Wunneeded-internal-declaration -Werror -Wno-unknown-cuda-version  %s -o %t.out
```

Comment:
What about "-pedantic" ? Should we add this option?

Reply:
What is "-pedantic"?